api/api.go

```diff
@@ -314,6 +315,7 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 		apiAlert := &APIAlert{
 			Alert:  &a.Alert,
 			Status: status,
+			ID:     fmt.Sprintf("%d", a.Fingerprint()),
```

Comment:
I am sorry for the wrong answer here @stuartnelson3. Yes, changing to base16 is due to reducing the length. In addition we prevent someone of interpreting the fingerprint as an increasing integer, and we enforce exposing the fingerprint as a string in a JSON data structure.